lib/not-found/not_found_screen.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
```

Comment:
"not-found" as a section seems out of place to me but I can PR for discussion later if I think of something

Reply:
it is a 404 page, and a page that will show errors related to navigation, while we build the app.  Moving it to another section is okay, I just didn't know where else to place it.